install_config/install/quick_install.adoc

```diff
@@ -24,15 +24,21 @@ the `atomic-openshift-installer` command, to install {product-title} across a
 set of hosts. This installer can deploy {product-title} components on targeted
 hosts by either installing RPMs or running containerized services.
 
+[IMPORTANT]
+====
+While RHEL Atomic Host is supported for running containerized {product-title}
+services, the installer is
+xref:../../install_config/install/host_preparation.adoc#software-prerequisites[provided
+by an RPM] and not available by default in RHEL Atomic Host. Therefore, it must
+be run from a Red Hat Enterprise Linux 7 system. The host initiating the
+installation does not need to be intended for inclusion in the {product-title}
+cluster, but it can be.
+====
+
```

Comment:
We'll provide a containerized installer in the future, but we won't start on that until after 3.6.